capstone.html

```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Electrical Engineering Portal</title>
+    <link rel="stylesheet" href="capstone.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css">
+</head>
+<header>
+    <nav class="container">
+        <div class="upper-nav">
+                <a href="https://www.facebook.com/haftamudesta"><i class="fa-brands fa-4x fa-facebook"></i></a> 
+            <i class="fa-brands fa-twitter-square"></i>
+            <p>English</p>
+            <p>My page</p>
+            <p>Logout</p>
+        </div>
+        <div class="lower-nav">
+            <div class="logo">
+                <a href="#home"><img src="images/electric-power-engineers1.jpg" alt=""></a>
+                <img src="images/Enabled.png" class="hamburger" alt="hambergur menu">
```

Comment:
- [ ] There appears to be a lot of extra space in the top Kindly make sure to remove it so that it matches the template. 

![image](https://user-images.githubusercontent.com/60618877/181587036-64e9be6a-4cf7-4f5a-8995-9b91bbd3e72d.png)